src/main/resources/plugin.yml

```diff
@@ -9,9 +9,15 @@ description: An expansion for Medieval Factions that allows faction members to c
 commands:
   fiefs:
     description: The main command for Fiefs.
```

Comment:
Perhaps we can extend this description to include subcommands? This is used in the /help text.

Reply:
Extended!